.yamato/upm-ci.yml

```diff
@@ -79,6 +85,40 @@ test_{{ platform.name }}_{{ editor.version }}:
 {% endfor %}
 {% endfor %}
 
+{% for editor in android_test_editors %}
```

Comment:
I think usually it's recommended to Split & Run for Android devices.

Since now, while the Editor is building your stuff you're allocating Android device, which is wasteful.

See https://github.cds.internal.unity3d.com/unity/mobile-yamato-example